engine/src/main/java/org/hibernate/validator/HibernateValidatorFactory.java

```diff
@@ -18,6 +20,16 @@
 public interface HibernateValidatorFactory extends ValidatorFactory {
 
 	/**
+	 * Returns a factory which is used to create {@link org.hibernate.validator.cfg.scriptengine.ScriptEvaluator}s for
```

Comment:
We can use non-qualified names.

Reply:
Why is btw. that this method is exposed here. What's is use case?

Reply:
It was done by analogy with the `ClockProvider` (for which there's `ValidatorFactory#getClockProvider()` ), so that I don't miss any places where this new ScriptEvaluator is needed. But looking at this method - it doesn't seems to be necessary. I'll remove it.